Beyond08-SOW.md

```diff
@@ -0,0 +1,485 @@
+# Red Hat Beyond 08
+
+Statement of Work (SOW)
+***
+
+## Pre-requirements
+
+You will be working on a project during this course.
+
+The following tech stack will be used.
+
+If you don't have knowledge/experience in any of them, it is worthwhile to look
+them up. The more you know the easier it will be.
+
+* Linux OS - able to use basic command line tools such as cd, ls, cat, grep,
+  less, vi, ...
+* Python - if/else, for loops, functions, classes
+* Django - create a basic or sample project
+* Git and GitHub - create a repo, commit, push, pull requests (PR), merge
+* HTML/CSS (Bootstrap) - design a basic web page from scratch, including using
+  CSS classes for basic formatting
+* Testing- unit tests and functional tests with pytest
+* Virtualization - Spin up a VM using Vagrant and VirtualBox
+* Databases - Basic familiarity with database concepts
+* Data modeling  
```

Comment:
Given that this is markdown, lines that are indented like this will look like source code. To nest bullet points you use `**`.

Reply:
Thanks 
his link this:
https://github.com/tchapi/markdown-cheatsheet/blob/master/README.md

I used `    *`
It looks good om my markdown tests server